Database.py

```diff
@@ -100,11 +100,11 @@ def insert_file(self):
                     if list==stat_list:
                         self.cursor_1.execute(stat_insert_query, element)
             ch_1.commit()
-            print("trying")
         except mysql.connector.Error as err: # try to find a more specific error 
             print(err) # maybe general database error. look for the type of expection
 
         try:
+            #list comprehension maybe
```

Comment:
In this case: maybe. It may work for calling the execute function multiple times for your queries, but usually list comp is used for *lists of things* and not for calling something a bunch of times.

I would stick with a standard for loop